packages/radix-ui-themes/src/components/text-field.css

```diff
@@ -368,14 +379,15 @@
     }
     &:where(:placeholder-shown) {
       cursor: var(--cursor-disabled);
```

Comment:
What's the logic behind only showing the disabled cursor if it's a placeholder and not if there's a value in the field?

Reply:
If there's a value in the field, you can still select and copy it, but when there's a placeholder, there's no interaction you can do.